src/platform/forms-system/test/js/review/submit-states/ValidationError.unit.spec.jsx

```diff
@@ -213,4 +213,53 @@ describe('Schemaform review: <ValidationError />', () => {
 
     tree.unmount();
   });
+
+  it('has the expected list of errors', () => {
+    const onBack = sinon.spy();
+    const onSubmit = sinon.spy();
+
+    const form = createForm();
+    form.formErrors = {
+      errors: [
+        {
+          name: 'test',
+          message: 'Missing test',
+          chapterKey: 'Test',
+          index: 0,
+        },
+        { name: 'zip', message: 'Zip', chapterKey: 'Zip', pageKey: 'zip' },
+        // No chapter = no link to open accordion
+        { name: 'empty', message: 'Property not found', chapterKey: '' },
+      ],
+    };
+
+    const formConfig = getFormConfig();
+    formConfig.showReviewErrors = () => true;
+
+    const formReducer = createformReducer({
+      formConfig: form,
+    });
+
+    const store = createStore();
+    store.injectReducer('form', formReducer);
+
+    const tree = render(
+      <Provider store={store}>
+        <ValidationError
+          appType="test"
+          buttonText="test"
+          formConfig={formConfig}
+          onBack={onBack}
+          onSubmit={onSubmit}
+          testId={'12345'}
+        />
+      </Provider>,
+    );
+    expect(tree.getByText(/missing some information/)).to.exist;
+    expect(tree.getByText(/information before you can submit/)).to.exist;
+    expect(tree.getByText('Missing test')).to.exist;
+    expect(tree.getByText('Zip')).to.exist;
+    expect(tree.getByText('Property not found')).to.exist;
+    tree.unmount();
+  });
 });
```

Comment:
Also need to test the behavior of passing in `formErrors`.

Reply:
Moved these tests into the ErrorLinks component.